engine/src/main/java/org/terasology/rendering/nui/widgets/UIList.java

```diff
@@ -44,6 +44,7 @@
     private final List<ItemInteractionListener> itemListeners = Lists.newArrayList();
     private final List<ItemActivateEventListener<T>> activateListeners = Lists.newArrayList();
     private final List<ItemSelectEventListener<T>> selectionListeners = Lists.newArrayList();
+    private Binding<Boolean> interactive = new DefaultBinding<>(true);
```

Comment:
It is not clear why we use DefaultBindings here to wrap around simple primitives.  I did it here only to be consistent.